packages/components/src/FormField/FormField.css

```diff
@@ -156,12 +160,16 @@ select.formField {
 }
 
 .miniLabel.large .label {
-  --field--padding-top: var(--space-small);
+  --field--padding-top: var(--space-smallest);
   --field--padding-bottom: var(--space-small);
   --field--padding-left: var(--space-large);
   --field--padding-right: var(--space-large);
 }
 
+.miniLabel .textareaLabel {
```

Comment:
Q: Why not just add this by default?
A: Because browsers likes to style things too

![image](https://user-images.githubusercontent.com/15986172/64284706-6e1b6f80-cf17-11e9-9be5-97a1f63a9e2a.png)